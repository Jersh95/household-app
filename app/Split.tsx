import {Text, View} from "react-native";

export default function Split() {
    return (
        <View>
            <Text>Split</Text>
        </View>
    )
}