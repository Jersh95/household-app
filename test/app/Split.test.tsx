import renderer from 'react-test-renderer';
import Split from "../../app/Split";

describe('Split', () => {
    it('should render', () => {
        const tree = renderer.create(<Split/>).toJSON();
        expect(tree).toMatchSnapshot();
    });
});