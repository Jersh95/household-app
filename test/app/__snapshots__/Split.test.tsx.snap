// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Split should render 1`] = `
<View>
  <Text>
    Split
  </Text>
</View>
`;
